app/src/main/java/com/example/lamnn/todo_mvp/data/repository/AsyncTaskTasksContent.java

```diff
@@ -0,0 +1,44 @@
+package com.example.lamnn.todo_mvp.data.repository;
+
+import android.os.AsyncTask;
+
+import com.example.lamnn.todo_mvp.data.model.Task;
+import com.example.lamnn.todo_mvp.data.source.Callback;
+
+import org.json.JSONException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AsyncTaskTasksContent extends AsyncTask<String, Void, List<Task>> {
+  private Callback<List<Task>> mCallback;
+  private Exception mException;
+
+  public AsyncTaskTasksContent(Callback<List<Task>> callback) {
+    mCallback = callback;
+  }
+
+  @Override
+  protected List<Task> doInBackground(String... strings) {
+    List<Task> tasks = new ArrayList<>();
+    try {
+      TasksDataAPI tasksDataAPI = new TasksDataAPI();
```

Comment:
1 tab = 4 space
e chỉnh lại config trong android studio
```suggestion
        TasksDataAPI tasksDataAPI = new TasksDataAPI();
```